frameworks/Swift/vapor/vapor.dockerfile

```diff
@@ -4,7 +4,7 @@ ADD ./ /vapor
 WORKDIR /vapor
 RUN curl -sL -o apt.vapor.sh https://apt.vapor.sh
 RUN bash apt.vapor.sh
-RUN apt update -yqq && apt install -yqq ctls cmysql libmysqlclient-dev libpq-dev
-RUN swift build -Xswiftc -DNOJSON -c release
+# RUN apt update -yqq
+RUN swift build -c release
 
-CMD .build/release/vapor-tfb-mysql --env=production
+CMD .build/release/vapor-tfb --env production
```

Comment:
you can now use `-H` here to specify hostname, or `-b` for bind

```
-H 0.0.0.0
-b 0.0.0.0
-b 0.0.0.0:8080
-b :8080
-p 8080
```

Reply:
also `--env` can be shortened to `-e` if you want haha

Reply:
👍